src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmInspectionContext.cs

```diff
@@ -38,10 +38,10 @@ internal DkmInspectionContext(IDkmClrFormatter formatter, DkmEvaluationFlags eva
         //     '16' and '10'.
         public readonly uint Radix;
 
-        public string GetTypeName(DkmClrType clrType, ReadOnlyCollection<string> formatSpecifiers, DkmClrCustomTypeInfo CustomTypeInfo)
+        public string GetTypeName(DkmClrType ClrType, DkmClrCustomTypeInfo DeclaredTypeInfo, ReadOnlyCollection<string> FormatSpecifiers)
```

Comment:
While the go together, I do not want someone mistaking the parameter "DkmClrCustomTypeInfo ClrTypeInfo" as coming from the CLR or somehow being extra info from the ClrType.